### DISCUSSION TITLE
Logrotation - how to check if it has run?

### DISCUSSION THREAD

**Discussion by ameinild** [NginxProxyManager/nginx-proxy-manager#1264] [Q&A] [answered]
With v. 2.9.5, default logrotation should be enabled. However, I'm having a hard time finding out if it has actually run on all available files. How do I check if logrotation has run in the container?

In the logs directory, only 1 single `fallback_error.log` has been rotated, none of the `proxy_host-X.log` or `redirection_host-X.log` files have been rotated, although they are much larger.

![Wetty](https://user-images.githubusercontent.com/434479/127311059-e0578597-5bf5-433d-8086-acb32edaa797.png)

So how do you confirm if this is actually working as it is supposed to? Thanks in advance.

**Comment by chaptergy** [comment, marked as answer]
The filename of the logfiles has been changed to split between error and access logs, to keep them for a different amount of time. But in order to not interfere with existing setups this new filename is only used for new and updated hosts. And only those new log files are rotated. So in order for logrotate to kick in you need to update the host by just editing it and saving it again (you don't actually have to change anything)

**Reply by ameinild** [reply]
Thanks for the clarification - this works perfectly as expected. 😊

Perhaps this could be specified somewhere in the release notes or in the readme, maybe I'm not the only user who wasn't aware of this little trick?
